src/core/server/models/user/user.ts

```diff
@@ -716,40 +764,48 @@ export async function setUserUsername(
 }
 
 /**
- * updateUserUsername will set the username of the User.
+ * updateUsername will set the username of the User.
  *
  * @param mongo the database handle
  * @param tenantID the ID to the Tenant
  * @param id the ID of the User where we are setting the username on
  * @param username the username that we want to set
+ * @param createdBy the user making the change
  */
-export async function updateUserUsername(
+
+export async function updateUsername(
```

Comment:
The function name should remain `updateUserUsername` as with other model scoped functions.